### DISCUSSION TITLE
#map global howto ?

### DISCUSSION THREAD

**Discussion by fal3nsh** [scandum/tintin#82] [Script help]
Hi,

I don't get how to set up and use #map global <vnum>. If I set it up with my recall location room, I get trouble using the map hereafter... I miss the point. Could you provide some basic example, please ? Thanks.

**Comment by scandum** [comment]
You'd want to create a new room, easiest is to use `#map goto {<free vnum>} dig`, set #map global to its vnum, then inside the global room use: `#map link {recall} {<vnum of recall room>}`.

If there are other global transportation commands you can add additional exits to the global room to handle those.

**Reply by fal3nsh** [reply]
Wow, finally got it. Thank you again. I was totally missing the point about first having a special dedicated "global" room... I've been trying to assign all this to my recall location.

This little example finds maybe its place once in the tintin++ manual :)
